nifi-nar-bundles/nifi-azure-bundle/nifi-azure-processors/src/main/java/org/apache/nifi/processors/azure/eventhub/GetAzureEventHub.java

```diff
@@ -64,8 +64,12 @@
 import org.apache.nifi.processors.azure.eventhub.utils.AzureEventHubUtils;
 
 @Tags({"azure", "microsoft", "cloud", "eventhub", "events", "streaming", "streams"})
-@CapabilityDescription("Receives messages from Microsoft Azure Event Hubs, writing the contents of the Azure message to the content of the FlowFile. "
-        + "Note: Please be aware that this processor creates a thread pool of 4 threads for Event Hub Client. They will be extra threads other than the concurrent tasks scheduled for this processor.")
+@CapabilityDescription("Receives messages from Microsoft Azure Event Hubs, writing the contents of the message to the content of the FlowFile. "
+        + "Note: Please be aware that this processor creates a thread pool for Event Hub Client. "
+        + "They will be extra threads other than the concurrent tasks scheduled for this processor. "
+        + "Please also be aware that this processor does not support checkpoints. "
+        + "Meaning there is no way to continue receiving messages from a certain offset after the processor has been stopped, "
+        + "which can lead to data duplication. It is recommended to use ConsumeAzureEventHub instead if possible.")
```

Comment:
With the suggested rewording in the first sentence, recommend the following changes:
```suggestion
        + "Lack of message checkpointing in GetAzureEventHub can result in data loss or data duplication depending on scheduling settings.")
```